google-cloud-pubsub/test/google/cloud/pubsub/topic/subscribe_test.rb

```diff
@@ -23,7 +23,7 @@
   it "creates a subscription when calling subscribe" do
     create_res = Google::Cloud::PubSub::V1::Subscription.new subscription_hash(topic_name, new_sub_name)
     mock = Minitest::Mock.new
-    mock.expect :create_subscription, create_res, [subscription_path(new_sub_name), topic_path(topic_name), push_config: nil, ack_deadline_seconds: nil, retain_acked_messages: false, message_retention_duration: nil, labels: nil, enable_message_ordering: nil, dead_letter_policy: nil, options: default_options]
+    mock.expect :create_subscription, create_res, [subscription_path(new_sub_name), topic_path(topic_name), push_config: nil, ack_deadline_seconds: nil, retain_acked_messages: false, message_retention_duration: nil, labels: nil, enable_message_ordering: nil, dead_letter_policy: nil, retry_policy: nil, options: default_options]
```

Comment:
Will this line continue to grow as we add more fields to the subscription? Maybe this is a Ruby styling thing I'm not aware of, but looks to me it's starting to get kind of unwieldy.

Reply:
It should be refactored. We have accumulated some technical debt in the unit tests.